docs/contributing/translations.md

```diff
@@ -68,8 +68,42 @@ _("Page {page_title} with status {status}".format(page_title=page.title, status=
 _(f"Page {page.title} with status {page.status_string}")
 ```
 
+For example (in template):
+  -   `some_tag` is a placeholder for any tag.
+  -   `arg_of_some_tag` is a placeholder for an argument of a tag.
+  -   `value_of_some_tag` is a placeholder for a value passed to a tag.
+  -   `filter` is a placeholder for a filter.
+
```

Comment:
No need for explanations of what tags and args are, the audience of this page are those that work Django regularly. But let's explain with a little intro a bit better.

```suggestion
### Translations with templates

You can import `i18n` and then translate with the `translate`/`blocktranslate` template tags. You can also translate string literals passed as arguments to tags and filters by using the familiar `_()` syntax.

```